apps/quickstart-integrations-monday-code/src/queue-service.js

```diff
@@ -0,0 +1,33 @@
+import { Logger } from "@mondaycom/apps-sdk";
+import { PubSub } from "@google-cloud/pubsub";
+const pubSubClient = new PubSub();
+
+const logTag = "QueueService";
+const logger = new Logger(logTag);
+
+
+export const produceMessage = async ({ body }) => {
+    const topicName = process.env.MNDY_TOPIC_NAME
+    logger.info(`produce message received ${JSON.stringify(body)}`);
+    logger.info(`about to publish to topic "${topicName}"`);
+    const dataBuffer = Buffer.from(JSON.stringify(body));
+    const messageId = await pubSubClient
+        .topic(topicName)
+        .publishMessage({ data: dataBuffer, attributes: { "Content-Type": "application/json" } });
+    logger.info(`Message ${messageId} published.`);
+    return messageId;
+}
+
+export const readQueueMessage = ({ body, query }) => {
```

Comment:
Consider adding these functions to the SDK, eg: 

```
const PubSub = new MondayPubSub()
PubSub.publish({body})
PubSub.readMessage({body, query})
```

Reply:
@yuhgto 

`PubSub.readMessage({body, query})` is not a thing as the server is "serverless". It's not up and running all the time and have the capability to start reading messages from a queue. The solution is to have this endpoint we're triggering on the service whenever there is a new message in the queue. The endpoint we're triggering is:
`/mndy-queue`